spec/services/stats/warehouse/case_report_sync_spec.rb

```diff
@@ -191,7 +207,7 @@
       let(:new_team_name) { Faker::Company.name }
 
       before do
-        Warehouse::CaseReport.generate(responded_case)
+        ::Warehouse::CaseReport.generate(responded_case)
```

Comment:
Just wondering, what's the purpose of this `::` here at the start of the line? I don't think I've seen that before

Reply:
Ensures module namespace is traversed from root rather than the current class/module. I found semaphore builds kept failing without the :: from root whereas locally it didn't